### PR TITLE
share file to multiple users

### LINKED ISSUE #50: share file to multiple user at a time
`share` function should able to take email of multiple users and share the file with all of them.

### PR DESCRIPTION
**Pull request by raghav-dalmia**
Screenshot from 2019-04-15 23-36-45
Fixes #50 
@nurdtechie98 @inishchith please review this PR.

### CODE REVIEW

**Comment by nurdtechie98** [comment]
@raghav-dalmia make the PR to dev branch there are unnecessary comits by changing the base.

**Comment by raghav-dalmia** [comment]
done @nurdtechie98, please check code owner file once before merging. if requires any changes please let me know.

**Comment by raghav-dalmia** [comment]
ping @nurdtechie98 @inishchith

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1 @@
-
-*       @nurdtechie98 @shivam1708 @inishchith @NightWing1998
\ No newline at end of file
+*       @nurdtechie98 @shivam1708 @inishchith @NightWing1998
```
Comment (inishchith) [inline .github/CODEOWNERS]: I'm not sure if this change is required.

Comment (raghav-dalmia) [reply .github/CODEOWNERS]: I also not sure about that, it is creating merge conflict. That's why I said check CODEOWNERS file. Chirag please confirm.

Comment (nurdtechie98) [reply .github/CODEOWNERS]: yes I did that

**Comment by nurdtechie98** [comment]
LGMT @raghav-dalmia Good Job. Merging now

**Pull request merged**